docs/UserGuide.md

```diff
@@ -1033,16 +1114,23 @@ Customer and order data are saved in the hard disk automatically after any comma
 
 All data are saved as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
 
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Notes about file name:**<br>
+
+* If you are confused why the file name is addressbook.json, this is because LoyaltyLift was built on top of an existing application called addressbook.
+</div>
```

Comment:
Are we not changing the filename to `loyaltylift.json`?

Reply:
I was initially deciding to leave it as addressbook but i think now i managed to find the spot to change to loyaltylift.json